packages/html/stories/Wires.stories.js

```
@@ -153,7 +153,7 @@ const Template = ({ label, ...args }) => {
         if (state.cell.isEdge()) {
           let style = this.view.getEdgeStyle(state);
 
-          if (style == EdgeStyle.WireConnector) {
+          if (style == WireConnector) {
```

ESLint

[error] 'WireConnector' is not defined.

(no-undef)